backend/routes.js

```diff
@@ -166,3 +167,12 @@ route.GET('/file/{hash}', {}, function (request, h) {
     return Boom.notFound()
   }
 })
+
+route.GET('/translations/get/{language}', {}, function (request, h) {
+  try {
+    return sbp('backend/translations/get', request.params.language)
```

Comment:
You defined `'backend/translations/get'` as `async` so it must be treated that way here too.

Reply:
That was for consistency with `route.GET('/file/{hash}'` where the return statement does not use `await` either, although the returned value is a Promise.

Reply:
Hmm. Fair point. Both should be consistent. I think both versions will work because I'm pretty sure Hapi handles returning an error when a promise is returned that is rejected. I just think it means that if there's an error in the promise, the error handler that is run is Hapi's and not ours, so in that case both this and the implementation of `route.GET('/file/{hash}'` might need updating, but it's no big deal. We can ignore this for now.

Reply:
Ah, I see you updated it. That's great! Leave it as-is with the latest commit you made. Great job! This will remind us to deal with `/file/{hash}` too